docs/UPGRADING.rst

```diff
@@ -34,3 +35,38 @@ This file describes the procedure for upgrading HIL to a new version.
 
      $ systemctl start httpd
      $ systemctl start hil_network
+
+Release Notes
+-------------
+
+0.3 (Upcoming)
```

Comment:
Upcoming is a bit vague to me. Maybe having an actual date?

Reply:
we don't really have an actual date. it's just as soon as we are done with all our issues marked with 0.3 milestone.

Reply:
sure, am working on the version PR, which is the last PR for 0.3 milestones I believe